CHANGELOG.md

```diff
@@ -224,6 +247,14 @@ showing the source changes from the previous version.
 
 ## [0.12.0](https://github.com/opendp/opendp/compare/v0.11.1...v0.12.0) - 2024-12-19
 
+### Migration
+
+Previously, OpenDP allowed arbitrary functions to be used as postprocessors. Unfortunately this could be abused to build measurements whose postprocessors introduce vulnerabilities. For instance, a post-processor that reads the system time could be used to add timing information to the output of a mechanism. OpenDP now requires `honest-but-curious` to be enabled in order to build postprocessors from Python, and states necessary conditions for valid postprocessors.
+
+Also:
+- Rename `SMDCurve` to `PrivacyProfile`
+- Remove argument for distance type on privacy measures: `max_divergence(T=float)` is now just `max_divergence()`
```

Comment:
- `SMDCurve` represented the function `ε(δ)`, and the new `PrivacyProfile` represents the function `δ(ε)`.